File: atv1/atv1.py
#objetivo é fazer um código que inverta os numeros dentro de um vetor.
vetor = [1,2,5,7,9,0]
vetorInvertido = []
tamanho = len(vetor)

for i in range(len(vetor)):
    print(tamanho)
    tamanho = tamanho- 1
    vetorInvertido.append(vetor[tamanho])
    

print(vetor,vetorInvertido)

File: atv1/atv1.c
#include<stdio.h>

int main() {
    int vetor[5] = {1,2,5,9,0};
    int vetorInvertido [sizeof(vetor)/sizeof(vetor[0])];
    int i;
    int tamanho =sizeof(vetor)/sizeof(vetor[0]);
    printf("%d",tamanho);
    printf("\n");
    for (i = 0; i < tamanho;i++){
    vetorInvertido[tamanho - i -1] = vetor[i];
        
    }
    printf("\n");
    for (i=0;i< tamanho;i++){
    printf("%d",vetor[i]);
    }
    printf("\n");
    for (i=0;i< tamanho;i++){
    printf("%d",vetorInvertido[i]);
    }
}
